### PR TITLE
Using absolute (alias) imports

### PR DESCRIPTION
**Pull request by uidoyen**
Using absolute (alias) component import to avoid dot-dot-slashes
Rename `appHeader.js` to `AppHeader`

### CODE REVIEW

File: ui/packages/ui/src/app/components/connectorStepHelpers/FilterTreeComponent.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ import {
 import { CubesIcon, ExclamationCircleIcon } from "@patternfly/react-icons";
 import React from "react";
 import { PageLoader } from "..";
-import { ApiError } from "../../shared";
-import { WithLoader } from "../../shared/WithLoader";
+import { ApiError } from "shared";
+import { WithLoader } from "shared/WithLoader";
```
Comment (mdrillin) [inline ui/packages/ui/src/app/components/connectorStepHelpers/FilterTreeComponent.tsx:14]: I think ApiError and WithLoader can be combined into one import, if WithLoader is added to index.ts in shared.  There are probably other similar cases in other components where imports could be combined with addition to index.ts

File: ui/packages/ui/src/app/pages/connectors/ConnectorsTableComponent.tsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ import { cellWidth, expandable, IAction, IExtraData, IRowData, Table, TableBody,
 import React, { SyntheticEvent } from "react";
 import isEqual from "react-fast-compare";
 import { useTranslation } from 'react-i18next';
-import { PageLoader, ToastAlertComponent, ConnectorIcon } from "../../components";
+import { PageLoader, ToastAlertComponent, ConnectorIcon } from "components";
 import { AppLayoutContext } from '../../Layout/AppLayoutContext';
-import { ApiError, ConfirmationButtonStyle, ConfirmationDialog, ConfirmationType, ConnectorTypeId, fetch_retry } from "../../shared";
-import { WithLoader } from "../../shared/WithLoader";
+import { ApiError, ConfirmationButtonStyle, ConfirmationDialog, ConfirmationType, ConnectorTypeId, fetch_retry } from "shared";
+import { WithLoader } from "shared/WithLoader";
```
Comment (indraraj) [inline ui/packages/ui/src/app/pages/connectors/ConnectorsTableComponent.tsx:29]: here

File: ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx [hunk 3]
```diff
@@ -24,8 +24,8 @@ import _ from "lodash";
 import React, { Dispatch, ReactNode, SetStateAction } from "react";
 import { useTranslation } from "react-i18next";
 import { Prompt } from "react-router-dom";
-import { ToastAlertComponent } from "../../components";
-import { ConnectionPropertiesError } from "../../components/ConnectionPropertiesError";
+import { ToastAlertComponent } from "components";
+import { ConnectionPropertiesError } from "components/ConnectionPropertiesError";
```
Comment (indraraj) [inline ui/packages/ui/src/app/pages/createConnector/CreateConnectorComponent.tsx:28]: here

File: ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConnectorTypeStep.tsx [hunk 4]
```diff
@@ -11,10 +11,10 @@ import {
...
 import { ExclamationTriangleIcon } from "@patternfly/react-icons";
 import React from "react";
-import { PageLoader, ConnectorIcon } from "../../../components";
-import { ApiError } from "../../../shared";
-import { getConnectorTypeDescription } from "../../../shared/Utils";
-import { WithLoader } from "../../../shared/WithLoader";
+import { PageLoader, ConnectorIcon } from "components";
+import { ApiError } from "shared";
+import { getConnectorTypeDescription } from "shared/Utils";
+import { WithLoader } from "shared/WithLoader";
```
Comment (indraraj) [inline ui/packages/ui/src/app/pages/createConnector/connectorSteps/ConnectorTypeStep.tsx:17]: here

File: ui/packages/ui/src/app/pages/createConnector/connectorSteps/FilterConfigStep.tsx [hunk 5]
```diff
@@ -12,17 +12,17 @@ import {
...
 import React, { SetStateAction } from "react";
 import { useTranslation } from "react-i18next";
-import { FilterTreeComponent } from "../../../components";
-import { FilterExcludeFieldComponent } from "../../../components/connectorStepHelpers/FilterExcludeFieldComponent";
-import { FilterInputFieldComponent } from "../../../components/connectorStepHelpers/FilterInputFieldComponent";
-import { NoPreviewFilterField } from "../../../components/connectorStepHelpers/NoPreviewFilterField";
+import { FilterTreeComponent } from "components";
+import { FilterExcludeFieldComponent } from "components/connectorStepHelpers/FilterExcludeFieldComponent";
+import { FilterInputFieldComponent } from "components/connectorStepHelpers/FilterInputFieldComponent";
+import { NoPreviewFilterField } from "components/connectorStepHelpers/NoPreviewFilterField";
```
Comment (indraraj) [inline ui/packages/ui/src/app/pages/createConnector/connectorSteps/FilterConfigStep.tsx:18]: this one too can be imported from 'components/connectorStepHelpers'

**Comment by indraraj** [comment]
Hey @uidoyen apart from merging multiple imports, Pr looks fine to me. And thanks for making this alias import I think it's a good improvement and makes our code looks cleaner :+1:.

**Review by indraraj — COMMENTED** [review]
Hey @uidoyen apart from merging multiple imports, Pr looks fine to me. And thanks for making this alias import I think it's a good improvement and makes our code looks cleaner :+1: .

**Comment by uidoyen** [comment]
Changes has been taken care in the last commit. Thanks!

**Review by indraraj — APPROVED** [review]

**Review by mdrillin — APPROVED** [review]
Looks good - thanks!

**Pull request merged**